bundle/language_en.properties

```diff
@@ -1442,6 +1442,7 @@ angal.therapy.patientvisitssaved
 angal.therapy.period                                                                                   = Period
 angal.therapy.pharmaceutical                                                                           = Pharmaceutical
 angal.therapy.pleaseinsertaquantitygreaterthanzero                                                     = Please insert a quantity greater than 0
+angal.therapy.pleasesavethevisitplanfirst.msg                                                          = Please save the visits plan first
```

Comment:
This is a complete sentence so it should end in a period.

Reply:
You mean only in English or any language?

Reply:
I would assume in any language as it is a complete sentence and not just a fragment.

Reply:
So there are others, like the one in 1444

Reply:
Done for both so far